Dockerfile

```diff
@@ -1,111 +1,29 @@
-    chmod 4755 /usr/local/bin/fixuid && \
-    mkdir -p /etc/fixuid && \
-    printf "user: $USER\ngroup: $GROUP\npaths:\n  - /service\n" > /etc/fixuid/config.yml
-
-ARG VCSREF
-ARG VERSION
-
-LABEL org.label-schema.schema-version="1.0" \
-      org.label-schema.vcs-url="https://github.com/hioa-cs/includeos" \
-      org.label-schema.vendor="IncludeOS" \
-      org.label-schema.vcs-ref=$VCSREF \
-      org.label-schema.version=$VERSION
-
-#########################
-FROM base as source-build
-
-RUN apt-get update && apt-get -y install \
     git \
-    lsb-release \
-    net-tools \
-    wget \
     && rm -rf /var/lib/apt/lists/*
 
-RUN mkdir -p /root/IncludeOS
-WORKDIR /root/IncludeOS
-COPY . .
-
-# Ability to specify custom tag that overwrites any existing tag. This will then match what IncludeOS reports itself.
-ARG VERSION
-RUN git describe --tags --dirty > /ios_version.txt
-RUN echo ${VERSION:=$(git describe --tags --dirty)} && git tag -d $(git describe --tags); git tag $VERSION && git describe --tags --dirty > /version.txt
-
-# Installation
-RUN ./install.sh -n
-
-#############################
-FROM base as grubify
-
-RUN apt-get update && apt-get -y install \
-  dosfstools \
-  grub-pc
-
-LABEL org.label-schema.description="Add a grub bootloader to any binary" \
-      org.label-schema.name="IncludeOS_grubify"
-
-COPY --from=source-build /usr/local/includeos/scripts/grubify.sh /home/ubuntu/IncludeOS_install/includeos/scripts/grubify.sh
-
-ENTRYPOINT ["fixuid", "/home/ubuntu/IncludeOS_install/includeos/scripts/grubify.sh"]
-
-###########################
-FROM base as build
-
-RUN apt-get update && apt-get -y install \
-    git \
-    clang-5.0 \
-    cmake \
-    nasm \
-    python-pip \
-    && rm -rf /var/lib/apt/lists/* \
-    && pip install pystache antlr4-python2-runtime && \
-    apt-get remove -y python-pip && \
-    apt autoremove -y
-
-# Metadata used for labels
-ARG BUILDDATE
-
-LABEL org.label-schema.description="Build a service using IncludeOS" \
-      org.label-schema.name="IncludeOS_builder" \
-      org.label-schema.docker.cmd="docker run -v $PWD:/service <container>" \
-      org.label-schema.build-date=$BUILDDATE
-
-WORKDIR /service
+# Install and configure Conan
+ENV conan_version=1_12_3
+RUN curl -Lo conan.deb https://dl.bintray.com/conan/installers/conan-ubuntu-64_$conan_version.deb \
+    && dpkg --install conan.deb \
+    && rm conan.deb
+RUN conan config install https://github.com/includeos/conan_config.git
 
-COPY --from=source-build  /usr/local/includeos /usr/local/includeos/
-COPY --from=source-build  /root/IncludeOS/etc/use_clang_version.sh /
-COPY --from=source-build  /root/IncludeOS/lib/uplink/starbase /root/IncludeOS/lib/uplink/starbase/
-COPY --from=source-build  /ios_version.txt /
-COPY --from=source-build  /version.txt /
-COPY --from=source-build  /root/IncludeOS/etc/docker_entrypoint.sh /entrypoint.sh
-ENTRYPOINT ["/entrypoint.sh"]
+# Configure build environment
+ENV CC=clang-6.0 \
+    CXX=clang++-6.0
```

Comment:
Maybe these could be extracted from conan profile so we don't have to hardcode them here? E.g. `conan profile get env.CXX [profile name]`